### PR TITLE
Updated example implementation and documentation for Line graph to be Carbon centric.

### LINKED ISSUE #164: Update documentation for Line component to be carbon-centric
# Story

## Description

As a current consumer of carbon, I'd like for the public facing documentation for the Line component to utilize carbon until terra-graphs is released

## Motivation

So that I will have the information I need to use the Line component from carbon as things currently are released

## Definition of Done

We'll know we're done when the documentation in terra-graphs-docs is utilizing carbon instead of terra-graphs.

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

- Made changes to example implementation and documentation for all line graph examples to make the dev site carbon centric.

Closes #164 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Testing

- All existing tests are successful.

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

**Comment by sdadn** [comment]
Instead of updating `packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs`, I believe we should create a new folder `packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon` and put Carbon-centric graphs there. This will prevent the need to undo this PR when we are ready to showcase react based examples.

**Comment by AshishMotanamGurunadham** [comment]
> Instead of updating `packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs`, I believe we should create a new folder `packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon` and put Carbon-centric graphs there. This will prevent the need to undo this PR when we are ready to showcase react based examples.

Fixed in this [commit](https://github.com/cerner/terra-graphs/pull/180/commits/f2a7650e6c947462ba96579e6cebce9f4ec4f80d).

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/LineGraph/BasicLineGraphs/DashedLine.6.doc.mdx [hunk 1]
```diff
@@ -10,19 +9,16 @@ This is a Simple graph with dashed line in it.
...
-  - `npm install terra-graphs`
+  - `npm i @cerner/carbon-graphs --save-dev`
 
 ## Usage
-```jsx
-import React from 'react';
-import LineGraph from '@cerner/terra-graphs/lib/components/Line/LineGraph';
+```js
+import Carbon from "@cerner/carbon-graphs/dist/js/carbon-graphs.js";
+import "@cerner/carbon-graphs/dist/css/carbon-graphs.css";
```
Comment (benbcai) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/LineGraph/BasicLineGraphs/DashedLine.6.doc.mdx:17]:
```suggestion
import Carbon from '@cerner/carbon-graphs/dist/js/carbon-graphs.js';
import '@cerner/carbon-graphs/dist/css/carbon-graphs.css';
```

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/LineGraph/BasicLineGraphs/DashedLine.6.doc.mdx]: Fixed in this [commit](https://github.com/cerner/terra-graphs/pull/180/commits/69443571d46a4de8c3bc435a0c403d57c3203f98).

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/LineGraph/BasicLineGraphs/Y2Axis.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  }, []);
+  return (
+    <React.Fragment>
+      <div id="tooltip" className="initial-tooltip" />
+      <div id="lineY2Axis" />
+    </React.Fragment>
+  );
+};
+
+export default LineExample;
```
Comment (benbcai) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/LineGraph/BasicLineGraphs/Y2Axis.jsx:32]: Not a big deal but typically the export module name would match the file name (in this case Y2Axis) unless there are multiple exports. I'm seeing that we use the same `LineExample` for all of these examples. This is probably not an issue but just a bit inconsistent with naming we have been using.

**Review by benbcai — APPROVED** [review]

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/LineGraph/BasicLineGraphs/Y2Axis.jsx (hunk 2 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/LineGraph/BasicLineGraphs/Y2Axis.jsx]: Fixed in [this](https://github.com/cerner/terra-graphs/pull/180/commits/54b52861307d9ef55bc7df64ff395982154d707a) commit.

File: packages/terra-graphs/CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,6 @@
   * Added reflow to the Gantt component.
 
 * Changed
-  * Made changes to WDIO test url's according to terra rules.
+  * Made changes to WDIO test url's according to terra rules.
```
Comment (sdadn) [inline packages/terra-graphs/CHANGELOG.md:22]: Is this change adding an extra space? If so, I would remove it.

**Review by sdadn — APPROVED** [review]

File: packages/terra-graphs-docs/CHANGELOG.md [hunk 4]
```diff
@@ -20,3 +20,4 @@
 
 * Changed 
   * Moved core documentation to terra-dev-site.
+  * Changed example implementation and documentation for Line graph to be carbon-centric.
```
Comment (sdadn) [inline packages/terra-graphs-docs/CHANGELOG.md:23]: nit: `Updated Line graph documentation and examples to be carbon-centric.`

File: packages/terra-graphs/CHANGELOG.md (hunk 3 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs/CHANGELOG.md:22]: I made a test change in line graph, that is the reason why i added a space here.

File: packages/terra-graphs-docs/CHANGELOG.md (hunk 4 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/CHANGELOG.md]: Fixed [here](https://github.com/cerner/terra-graphs/pull/180/files#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1R23).

**Review by tbiethman — APPROVED** [review]

**Pull request merged**